src/actions/Kangaroo/CmdbKangaroo.ts

```diff
@@ -0,0 +1,79 @@
+/*-
+ * #%L
+ * cmdb
+ * %%
+ * Copyright (C) 2021 TODO: Enter Organization name
+ * %%
+ * TODO: Define header text
+ * #L%
+ */
+import { CmdbBase } from "../CmdbBase";
+import { invokeWorkflow } from "../invokeWorkflow";
+export class CmdbKangaroo extends CmdbBase {
+    settings(){
+        this.httpStatusOK = {min: 200, max: 200};
+        this.httpStatusFail = {min: 400, max: 400};
+    }
+
+    /**
+     * 
+     * @returns 
+     */
+    public Add() {
+
+        let wfId = 'A18080808080808080808080808080808080808001299080088268176866967b3'; // Worflow "Invoke a REST operation"
+        let restContent = JSON.stringify({              // html body
+            name : this.recordName,
+            size : this.recordSize
+        })
+        let restAcceptHeaders = ['*/*'];
+        let inputs:Record<string, any> = {
+            content: restContent,
+            acceptHeaders: restAcceptHeaders,
+            defaultContentType: "application/json",
+        }
+        let settings:Record<string, any> = {
+            type: 'REST',                                       // Type of Workflow
+            restHostID: '45767ca5-a644-4a66-8f78-d6dcc257aa07', // REST Host "kangaroo"
```

Comment:
As a best practise, try to not hardcode IDs of vRO inventory objects - those change from one environment to another. Put them in a configuration element instead